js/script.js

```diff
@@ -38,6 +38,14 @@ const updateTask = (task, newTitle) => {
   renderTasks();
 };
 
+const doneTask = (taskId) => {
+  const task = tasks.find((task) => task.id === taskId);
+  if (task) {
+    task.done = !task.done;
```

Comment:
still seems to toggle the Done state, is it what the story requirement says?

Reply:
Bhai, sorry for overlook the issue, now it is resolved.